specification/common/AzureResource.json

```diff
@@ -0,0 +1,41 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "AzureResource": {
```

Comment:
What's the reason of adding these? We already have some of these common types defined here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json

Reply:
Fixed. And move the AzureEntityResource to type.json.